packages/web-components/src/components/rux-switch/rux-switch.scss

```diff
@@ -1,12 +1,11 @@
 @use '../../common/functional-components/FormFieldMessage/form-field-message.scss';
 :host {
     display: block;
-    padding-left: 1%;
 }
 .rux-switch {
     position: relative;
     display: flex;
-    height: 22px;
+    height: var(--line-height-sm);
```

Comment:
can we keep this hardcoded at 22px for now since its really a size token? if design changes `line-height-sm` (and they wont any time soon), it might unintended side effects. a more realistic scenario would be we change our line height tokens to be unitless values, which they should be but we're constrained at the moment.

Reply:
Sure! So it's actually 20px so to bring it inline with my other height 20px calculations I did a `calc(var(--spacing-4) + var(--spacing-1))` I figure if I'm consistent it'll be easier to find in future. Is that ok?